packages/css/src/components/index.scss

```diff
@@ -45,7 +45,6 @@
 @use "logo/logo";
```

Comment:
I found another mega menu remnant [here](https://github.com/Amsterdam/design-system/blob/develop/storybook/src/pages/amsterdam/common/AppHeader.tsx#L28)

Reply:
I’ve added ‘header’ into the mega menu id and removed the div that used to apply CSS Columns.

Reply:
I wouldn't add changes to Header to this PR. This PR is for deprecating Mega Menu.

Reply:
True, removed.